imap_processing/cdf/config/shared/hermes_default_global_cdf_attrs_schema.yaml

```diff
@@ -0,0 +1,278 @@
+CDF_Lib_version:
+  description: > 
+    Version of the CDF Binaries library used to generate the CDF File
+  default: null
+  derived: true
```

Comment:
You may also want to remove the derived keys here if you're not going to auto-generate any of the metadata attributes. 

I was thinking of _not_ including these in the "generalized" version of the schema, and only including them in sub-classed versions of the schema.